src/components/Drift/index.js

```diff
@@ -0,0 +1,176 @@
+                  fill="rgb(255, 255, 255)"
+                  fill-rule="evenodd"
+                />
+              </svg>
+            ) : icon === "D" ? (
+              <svg
+                width="25"
+                height="25"
+                viewBox="0 0 25 25"
+                focusable="false"
+                aria-hidden="true"
+                style={{
+                  display: state === STATES.INITIAL ? "initial" : "none"
+                }}
+              >
+                <path
+                  d="M23.295 24.412a.987.987 0 0 1-1.123-.43L19.9 20.358H9.527c-1.392 0-2.523-1.153-2.523-2.57V16.3c0-.563.448-1.02 1-1.02.553 0 1 .457 1 1.02v1.485c0 .295.235.533.523.533h10.92c.036 0 .067.018.103.022a.96.96 0 0 1 .527.224.984.984 0 0 1 .14.15c.023.026.054.043.073.075l.725 1.154V9.7a.528.528 0 0 0-.523-.532h-1.48c-.55 0-1-.457-1-1.02 0-.56.45-1.018 1-1.018h1.48c1.392 0 2.524 1.152 2.524 2.57V23.433c0 .452-.293.85-.72.978zM14.49 13.226H4.116L1.844 16.85a.997.997 0 0 1-1.124.43 1.017 1.017 0 0 1-.72-.978V2.57C0 1.152 1.132 0 2.523 0H14.49c1.39 0 2.52 1.152 2.52 2.57v8.085c0 1.418-1.13 2.57-2.52 2.57zm.52-10.656a.528.528 0 0 0-.52-.532H2.522c-.288 0-.522.24-.522.532v10.242l.726-1.153c.02-.032.05-.05.07-.076a.897.897 0 0 1 .296-.256.92.92 0 0 1 .372-.118c.037-.004.068-.022.104-.022h10.92c.288 0 .522-.238.522-.533V2.57z"
+                  fill="rgb(255, 255, 255)"
+                  fill-rule="evenodd"
+                />
+              </svg>
+            ) : null}
+
+            <svg
+              width="17"
+              height="17"
+              viewBox="0 0 17 17"
+              focusable="false"
+              aria-hidden="true"
+              style={{
+                display: state === STATES.INITIAL ? 'none' : 'initial' 
+              }}
+            >
+              <path
+                d="M16.726 15.402c.365.366.365.96 0 1.324-.178.178-.416.274-.663.274-.246 0-.484-.096-.663-.274L8.323 9.648h.353L1.6 16.726c-.177.178-.416.274-.663.274-.246 0-.484-.096-.663-.274-.365-.365-.365-.958 0-1.324L7.35 8.324v.35L.275 1.6C-.09 1.233-.09.64.274.274c.367-.365.96-.365 1.326 0l7.076 7.078h-.353L15.4.274c.366-.365.96-.365 1.326 0 .365.366.365.958 0 1.324L9.65 8.675v-.35l7.076 7.077z"
+                fill="rgb(255, 255, 255)"
+                fill-rule="evenodd"
+              />
+            </svg>
+          </i>
+        </div>
+      </div>
+    </div>
+  )
+}
+
+Drift.propTypes = {
+  /**
+   * Change the style of the Drift messenger icon, one of the four default types available when setting up your Drift messenger or in the Drift Widget "Widget icon" settings. 
+   */
+  icon: PropTypes.oneOf(['A', 'B', 'C', 'D']).isRequired
```

Comment:
Might make sense to help enforce the props someone might use to configure the chat button?

Although, it might also be more preferable to build in typescript for extra enforcement. 

Can easily remove though!

Reply:
Given we provide a default prop I think we can remove the `isRequired`. Defining what we accept it great though 👍